WordPress/Classes/ViewRelated/Stats/Extensions/WPStyleGuide+Stats.swift

```diff
@@ -199,11 +199,11 @@ extension WPStyleGuide {
 
         struct PostingActivityColors {
             static let lightGrey = UIColor.neutral(shade: .shade10)
-            static let lightBlue = UIColor(fromRGBAColorWithRed: 145.0, green: 226.0, blue: 251.0, alpha: 1)
-            static let mediumBlue = UIColor(fromRGBAColorWithRed: 0.0, green: 190.0, blue: 246.0, alpha: 1)
-            static let darkBlue = UIColor(fromRGBAColorWithRed: 0.0, green: 131.0, blue: 169.0, alpha: 1)
-            static let darkGrey = UIColor.neutral(shade: .shade70)
-            static let orange = UIColor(fromRGBAColorWithRed: 245.0, green: 131.0, blue: 53.0, alpha: 1)
+            static let lightBlue = UIColor.muriel(color: MurielColor(name: .airo, shade: .shade10))
```

Comment:
Instead of accessing `airo` directly, use `brand` here. Also, I think `primary` (which maps to blue) is possibly the more correct color to use here, but that's up to @SylvesterWilmott

Reply:
Airo LGTM and also mirrors Calypso

Reply:
Maybe @porada could weigh in here for us, but I believe the intention for `brand` blue is to only be used for branding elements like our app icon and the top toolbar. Within the stats screen on dark mode we're exclusively using the new, more vibrant product blue and this screenshot @drw158 posted in another issue show that's the direction Calypso will be going as well.

![60832089-b53bed00-a180-11e9-8dbc-2e9cc30a008f](https://user-images.githubusercontent.com/1396552/60903772-f6470680-a23f-11e9-8bad-0cf16fde787f.png)

Reply:
> I believe the intention for brand blue is to only be used for branding elements like our app icon and the top toolbar.

Matt is right here. Please don’t use WordPress Blue (formerly Airo) for UIs. This will be one of our official color guidelines.

Reply:
@mattmiklic @porada @SylvesterWilmott this is a test using the primary blue. What do you think?

![Simulator Screen Shot - iPhone X - 2019-07-09 at 19 18 18](https://user-images.githubusercontent.com/912252/60909431-860ba580-a27e-11e9-9d53-04e132dd27b3.png)

Reply:
This looks correct, but will let Sylvester say if these are the specific shades he wants to use.

Reply:
I stand corrected! Let's roll with the blue. Thanks